javalin/src/main/java/io/javalin/http/staticfiles/PrecompressingResourceHandler.kt

```diff
@@ -0,0 +1,112 @@
+package io.javalin.http.staticfiles
+
+import io.javalin.core.compression.CompressionStrategy
+import io.javalin.core.util.Header
+import io.javalin.http.LeveledBrotliStream
+import io.javalin.http.LeveledGzipStream
+import org.eclipse.jetty.http.MimeTypes
+import org.eclipse.jetty.util.resource.Resource
+import java.io.*
+import javax.servlet.http.HttpServletRequest
+import javax.servlet.http.HttpServletResponse
+
+
+class PrecompressingResourceHandler(val compStrat: CompressionStrategy) {
```

Comment:
Is passing the `CompressionStrategy` required? The pre-compressor should always use maximum compression IMO, since it only does it 1 time.